packages/aragon-wrapper/src/index.js

```diff
@@ -175,10 +183,39 @@ export default class Aragon {
           ))
         )
       )
+    this.apps = this.appsWithoutIdentifiers
+      .withLatestFrom(
```

Comment:
One thing to note here (it might not work as expected): emitting events from `this.identifiers` won't trigger an update on `this.apps` so a new app would have to be emitted before any changes from new identifiers takes place.

Maybe something like `combineLatest` would be better?

Reply:
Emitting events from `this.identifiers` will indeed trigger an update, since it is the source observable. Look at the marble diagram [here]().

However, emitting from the `apps` observable will not. This is an overlook, and `combineLatest` would solve it.